app/views/play/level/tome/CastButtonView.coffee

```diff
@@ -73,7 +73,7 @@ module.exports = class CastButtonView extends CocoView
     castRealTimeShortcutVerbose + ': ' + $.i18n.t('keyboard_shortcuts.run_real_time')
 
   onCastButtonClick: (e) ->
-    Backbone.Mediator.publish 'tome:manual-cast', {}
+    Backbone.Mediator.publish 'tome:manual-cast', {cinematic: true}
```

Comment:
Does this mean cinematic is on everywhere?
Let's put this behind a feature flag and default to off for now.